buildSrc/src/main/groovy/com/uber/okbuck/core/model/android/AndroidLibTarget.groovy

```diff
@@ -22,4 +26,23 @@ class AndroidLibTarget extends AndroidTarget {
     ManifestMerger2.MergeType getMergeType() {
         return ManifestMerger2.MergeType.LIBRARY
     }
+
+    List<String> getKotlincArguments() {
+        ImmutableList.Builder<String> extraKotlincArgs = ImmutableList.<String>builder()
+        if (hasKotlinAndroidExtensions) {
+            StringBuilder options = new StringBuilder()
+            int index = 0
+            getResDirs().each {  String dir ->
+                // :root:module -> root/module/
+                String module = project.path.replace(":", File.separator).substring(1) + File.separator
+                String variant = getResVariants()[index++]
```

Comment:
Can you explain the logic here a bit?

how are we reasoning about the order from `getResDirs()` and `getResVariants()`?

It seems like they belong together in a data object almost?

Reply:
`Set` doesn't guarantee order so this isn't safe to do. I'll instead return the variant name and directory as a Tuple.